go/stellar/build.go

```diff
@@ -184,6 +299,17 @@ func BuildPaymentLocal(mctx libkb.MetaContext, arg stellar1.BuildPaymentLocalArg
 						Message: "Your Keybase username will not be linked to this transaction.",
 					})
 				}
+
+				offerAdvancedForm, err := bpc.ShouldOfferAdvancedSend(mctx, arg.From, arg.To)
+				if err != nil {
+					return res, err
```

Comment:
`BuildPaymentLocal` isn't allowed to error when given valid (including partial) inputs. But I think this would error on RPC. Instead, on error `log` and omit the banner.